Client/Frontend/Browser/Playlist/Playlist.swift

```diff
@@ -0,0 +1,277 @@
+// Copyright 2020 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import UIKit
+import CoreData
+import Shared
+import BraveShared
+
+private let log = Logger.browserLogger
+
+class Playlist {
```

Comment:
All of the `(try? self.backgroundContext.fetch(request))?` should be fixed to handle errors i.e. `do { } catch { log.error(error) }`